llvm/lib/Target/PowerPC/PPCISelDAGToDAG.cpp

```diff
@@ -6143,23 +6143,23 @@ void PPCDAGToDAGISel::Select(SDNode *N) {
            " ELF/AIX or 32-bit AIX in the following.");
 
     // Transforms the ISD::TOC_ENTRY node for 32-bit AIX large code model mode,
-    // or 64-bit medium (ELF-only), or large (ELF and AIX) code model code that
-    // does not conain TOC data symbols.
+    // 64-bit medium (ELF-only), or large (ELF and AIX) code model code that
```

Comment:
We discussed this offline. I was just misunderstanding the comment and read it wrong so the above can be disregarded. I think adding the `64-bit` before large would be helpful, though.
```suggestion
    // 64-bit medium (ELF-only), or 64-bit large (ELF and AIX) code model code that
```